poppy/poppy_core.py

```diff
@@ -2289,11 +2289,13 @@ def get_phasor(self, wave):
             else:
                 # raise NotImplementedError("Need to implement resampling.")
                 zoom = (self.pixelscale / wave.pixelscale).decompose().value
-                resampled_opd = scipy.ndimage.interpolation.zoom(self.opd, zoom,
-                                                                 output=self.opd.dtype,
+                original_opd = self.get_opd(wave)
```

Comment:
There are also some direct uses of `self.opd` in the `display()` function, which I think would similarly need to be changed to calls to `get_opd` to behave correctly with this kind of input.

Reply:
I grepped for `self.opd` but I don't see what you mean upon a cursory inspection...

Reply:
https://github.com/spacetelescope/poppy/blob/master/poppy/poppy_core.py#L2417

Reply:
Ok, just pushed a commit for that. @douglase just showed up in Tucson and he says hi 😄